2021-08-29 02:46:42,747 ERROR frappe New Exception collected with id: 2021-08-29 02:46:42.624855-78.100.211.126-977
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:46:43,228 ERROR frappe New Exception collected with id: 2021-08-29 02:46:43.224027-78.100.211.126-22d
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:47:09,131 ERROR frappe New Exception collected with id: 2021-08-29 02:47:09.063276-78.100.211.126-936
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:47:09,445 ERROR frappe New Exception collected with id: 2021-08-29 02:47:09.441046-78.100.211.126-4d4
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:53:43,977 ERROR frappe New Exception collected with id: 2021-08-29 02:53:43.906120-78.100.211.126-40d
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:53:44,509 ERROR frappe New Exception collected with id: 2021-08-29 02:53:44.505273-78.100.211.126-48b
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:54:03,172 ERROR frappe New Exception collected with id: 2021-08-29 02:54:03.105093-78.100.211.126-8bb
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:54:03,480 ERROR frappe New Exception collected with id: 2021-08-29 02:54:03.475652-78.100.211.126-2b1
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:54:31,009 ERROR frappe New Exception collected with id: 2021-08-29 02:54:31.005251-78.100.211.126-b99
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 02:54:31,393 ERROR frappe New Exception collected with id: 2021-08-29 02:54:31.388890-78.100.211.126-c1d
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 06:28:48,792 ERROR frappe New Exception collected with id: 2021-08-29 06:28:48.667202-78.100.211.126-a73
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 06:28:49,385 ERROR frappe New Exception collected with id: 2021-08-29 06:28:49.317713-78.100.211.126-221
Site: mum125.erpgulf.com
Form Dict: {}
2021-08-29 06:34:19,723 ERROR frappe New Exception collected with id: 2021-08-29 06:34:19.602834-78.100.211.126-d32
Site: mum125.erpgulf.com
Form Dict: {}
2021-09-02 06:09:33,790 ERROR frappe New Exception collected with id: 2021-09-02 06:09:33.657819-8.17.206.93-266
Site: mum125.erpgulf.com
Form Dict: {'key': '', 'old_password': 'Friday2000@T', 'new_password': '', 'logout_all_sessions': '1', 'cmd': 'frappe.core.doctype.user.user.update_password'}
2021-09-02 06:50:51,807 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","DocField"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,006 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","DocType Action"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,233 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Custom Field"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,433 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Property Setter"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,477 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","DocType Link"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,478 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","User"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 101, in process
    self.add_custom_fields()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 308, in add_custom_fields
    """, (self.name,), as_dict=1, update={"is_custom_field": 1})
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 145, in sql
    self._cursor.execute(query, values)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 148, in execute
    result = self._query(query)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 310, in _query
    conn.query(q)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 775, in _read_query_result
    result.read()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1163, in read
    self._read_result_packet(first_packet)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1236, in _read_result_packet
    self._read_rowdata_packet()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1270, in _read_rowdata_packet
    packet = self.connection._read_packet()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1412, 'Table definition has changed, please retry transaction')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,536 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","DocPerm"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 101, in process
    self.add_custom_fields()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 308, in add_custom_fields
    """, (self.name,), as_dict=1, update={"is_custom_field": 1})
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 145, in sql
    self._cursor.execute(query, values)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 148, in execute
    result = self._query(query)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/cursors.py", line 310, in _query
    conn.query(q)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 548, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 775, in _read_query_result
    result.read()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1163, in read
    self._read_result_packet(first_packet)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1236, in _read_result_packet
    self._read_rowdata_packet()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 1270, in _read_rowdata_packet
    packet = self.connection._read_packet()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/connections.py", line 725, in _read_packet
    packet.raise_for_error()
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/protocol.py", line 221, in raise_for_error
    err.raise_mysql_exception(self._data)
  File "/opt/bench/frappe-bench/env/lib/python3.6/site-packages/pymysql/err.py", line 143, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.OperationalError: (1412, 'Table definition has changed, please retry transaction')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,639 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","DocType Link"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,681 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","Role"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,683 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","DocType"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,762 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Web Form"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,769 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","=","Has Role"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,833 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Web Template"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,918 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Web Form Field"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:52,955 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Portal Menu Item"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
2021-09-02 06:50:53,003 ERROR frappe Could not take error snapshot: 'User' object has no attribute 'docstatus'
Site: mum125.erpgulf.com
Form Dict: {'doctype': 'DocType', 'fields': '["`tabDocType`.`name`","`tabDocType`.`owner`","`tabDocType`.`creation`","`tabDocType`.`modified`","`tabDocType`.`modified_by`","`tabDocType`.`_user_tags`","`tabDocType`.`_comments`","`tabDocType`.`_assign`","`tabDocType`.`_liked_by`","`tabDocType`.`docstatus`","`tabDocType`.`parent`","`tabDocType`.`parenttype`","`tabDocType`.`parentfield`","`tabDocType`.`idx`","`tabDocType`.`module`","`tabDocType`.`color`"]', 'filters': '[["DocType","name","like","%ART%"],["DocType","name","=","Data Migration Mapping Detail"]]', 'order_by': '`tabDocType`.`modified` desc', 'start': '0', 'page_length': '20', 'view': 'List', 'with_comment_count': 'true'}
Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 54, in application
    init_request(request)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/app.py", line 128, in init_request
    frappe.local.http_request = frappe.auth.HTTPRequest()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 32, in __init__
    self.set_session()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 69, in set_session
    frappe.local.login_manager = LoginManager()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 122, in __init__
    self.make_session(resume=True)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 206, in make_session
    full_name=self.full_name, user_type=self.user_type)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 189, in __init__
    self.resume()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/sessions.py", line 255, in resume
    validate_ip_address(self.user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/auth.py", line 390, in validate_ip_address
    user = frappe.get_cached_doc("User", user) if not frappe.flags.in_test else frappe.get_doc("User", user)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 807, in get_cached_doc
    doc = get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 850, in get_doc
    doc = frappe.model.document.get_doc(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 75, in get_doc
    return controller(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 113, in __init__
    self.load_from_db()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 142, in load_from_db
    if not getattr(self, "_metaclass", False) and self.meta.issingle:
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/base_document.py", line 91, in meta
    self._meta = frappe.get_meta(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 881, in get_meta
    return frappe.model.meta.get_meta(doctype, cached=cached)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 37, in get_meta
    meta = Meta(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 84, in __init__
    self.process()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 105, in process
    self.set_custom_permissions()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 420, in set_custom_permissions
    filters=dict(parent=self.name), update=dict(doctype='Custom DocPerm'))
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1433, in get_all
    return get_list(doctype, *args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/__init__.py", line 1406, in get_list
    return frappe.model.db_query.DatabaseQuery(doctype).execute(*args, **kwargs)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 101, in execute
    self.columns = self.get_table_columns()
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/db_query.py", line 338, in get_table_columns
    return get_table_columns(self.doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/meta.py", line 49, in get_table_columns
    return frappe.db.get_table_columns(doctype)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/database/database.py", line 894, in get_table_columns
    raise self.TableMissingError('DocType', doctype)
pymysql.err.ProgrammingError: ('DocType', 'Custom DocPerm')

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 36, in make_error_snapshot
    snapshot = get_snapshot(exception)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in get_snapshot
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/inspect.py", line 1269, in formatargvalues
    specs.append(convert(args[i]))
  File "/usr/lib64/python3.6/inspect.py", line 1266, in convert
    return formatarg(name) + formatvalue(locals[name])
  File "/opt/bench/frappe-bench/apps/frappe/frappe/utils/error.py", line 81, in <lambda>
    call = inspect.formatargvalues(args, varargs, varkw, locals, formatvalue=lambda value: '={}'.format(pydoc.text.repr(value)))
  File "/usr/lib64/python3.6/reprlib.py", line 55, in repr
    return self.repr1(x, self.maxlevel)
  File "/usr/lib64/python3.6/pydoc.py", line 1035, in repr1
    return cram(stripid(repr(x)), self.maxother)
  File "/opt/bench/frappe-bench/apps/frappe/frappe/model/document.py", line 1360, in __repr__
    docstatus = f" docstatus={self.docstatus}" if self.docstatus else ""
AttributeError: 'User' object has no attribute 'docstatus'
